governance/votes/Executive vote - July 29, 2022.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Template - [Executive Vote] Onboarding Real World Asset Vaults - July 29, 2022
+summary: Onboards Real World Asset Vaults for Huntingdon Valley Bank (HVBank) and Societe Generale – Forge (OFH).
+date: 2022-07-29T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Onboarding Real World Asset Vaults - July 29, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Amendments to the letter agreements between RWA Foundation and various parties for the RWA-009 (HVBank) legal construction. 
```

Comment:
Can we add a link to this forum post from Christian with a bit more context? https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/8?u=miguel_cabeza